content/en/docs/setup/release/notes.md

```diff
@@ -2441,7 +2441,7 @@ filename | sha512 hash
 - AppProtocol is a new field on Service and Endpoints resources, enabled with the ServiceAppProtocol feature gate. ([#88503](https://github.com/kubernetes/kubernetes/pull/88503), [@robscott](https://github.com/robscott)) [SIG Apps and Network]
 - BlockVolume and CSIBlockVolume features are now GA. ([#88673](https://github.com/kubernetes/kubernetes/pull/88673), [@jsafrane](https://github.com/jsafrane)) [SIG Apps, Node and Storage]
 - Consumers of the 'certificatesigningrequests/approval' API must now grant permission to 'approve' CSRs for the 'signerName' specified on the CSR. More information on the new signerName field can be found at https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/20190607-certificates-api.md&#35;signers ([#88246](https://github.com/kubernetes/kubernetes/pull/88246), [@munnerz](https://github.com/munnerz)) [SIG API Machinery, Apps, Auth, CLI, Node and Testing]
-- CustomResourceDefinition schemas that use `x-kubernetes-list-map-keys` to specify properties that uniquely identify list items must make those properties required or have a default value, to ensure those properties are present for all list items. See https://kubernetes.io/docs/reference/using-api/api-concepts/&#35;merge-strategy for details. ([#88076](https://github.com/kubernetes/kubernetes/pull/88076), [@eloyekunle](https://github.com/eloyekunle)) [SIG API Machinery and Testing]
+- CustomResourceDefinition schemas that use `x-kubernetes-list-map-keys` to specify properties that uniquely identify list items must make those properties required or have a default value, to ensure those properties are present for all list items. See https://kubernetes.io/docs/reference/using-api/api-concepts/#merge-strategy for details. ([#88076](https://github.com/kubernetes/kubernetes/pull/88076), [@eloyekunle](https://github.com/eloyekunle)) [SIG API Machinery and Testing]
```

Comment:
Wondering if we usually using code or direct character for something like #

Reply:
Direct character is the best solution for this, opening that link with a code in replacement for the normal # character doesnt work, browser see '&#35;' as '&#35;' within a loaded url, but if entered on search bar alone "&#35;" its treated as '#'

Reply:
Hi @saintmalik. Thanks for opening this pull request.
This page is automatically generated by the release team (SIG Release).
You can raise an issue with SIG Release (https://github.com/kubernetes/release#contributing) so that the release
notes tool is updated for the next release.
There are several links that are not correctly formatted.

Page preview:
https://deploy-preview-24567--kubernetes-io-master-staging.netlify.app/docs/setup/release/notes/#api-change-9

Original page:
https://kubernetes.io/docs/setup/release/notes/#api-change-9

Your change corrects the link. Another possible way of updating this page:


CustomResourceDefinition schemas that use `x-kubernetes-list-map-keys` to specify properties that uniquely identify list items must make those properties required or have a default value, to ensure those properties are present for all list items. See [API concepts](https://kubernetes.io/docs/reference/using-api/api-concepts/&#35;merge-strategy) for details. ([#88076](https://github.com/kubernetes/kubernetes/pull/88076), [@eloyekunle](https://github.com/eloyekunle)) [SIG API Machinery and Testing]

Reply:
nit: I think the link should be:
https://kubernetes.io/docs/reference/using-api/api-concepts/